Property_map/include/CGAL/property_map.h

```diff
@@ -49,18 +51,18 @@ class static_property_map
   V v;
 
 public:
-  static_property_map(V pv)
+  Static_property_map(V pv)
     :v(pv){}
   inline friend
   value_type
-  get(const static_property_map& pm, const key_type&)
+  get(const Static_property_map& pm, const key_type&)
   {
     return pm.v;
   }
 
   inline friend
   void
-  put(static_property_map& pm, const key_type&, const value_type&)
+  put(Static_property_map& pm, const key_type&, const value_type&)
   {}
 };
```

Comment:
This commit was not correctly tested. It triggers compilation errors, detected by Travis, [in Mesh_3][m3], and [in the Polyhedron demo][demo]. It seems to be the same culprit: two occurrences of the old name in `Surface_mesh/include/CGAL/boost/graph/PMP_properties_Surface_mesh.h`.

  [m3]: https://travis-ci.org/CGAL/cgal/jobs/233619359#L4138
  [demo]: https://travis-ci.org/CGAL/cgal/jobs/233619449#L5412